realm-transformer/src/main/groovy/io/realm/transformer/GroovyUtil.groovy

```diff
@@ -39,7 +38,7 @@ class GroovyUtil {
 
     @Nonnull
     static boolean isSyncEnabled(@Nonnull Project project) {
-        return project.realm?.syncEnabled != null && project.realm.syncEnabled
+        return (project?.hasProperty("realm")) ? project.realm.syncEnabled : false
```

Comment:
Why not simply `project.hasProperty("realm")` since `project` is marked as `Nonnull`?

Reply:
Yes, good point